lib/types.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import { Folder } from './files/folder'
+import { Node } from './files/node'
+import { View } from './navigation'
+
+type ActionContextSingle = {
+	nodes: [Node],
+	view: View,
+	context: Folder,
```

Comment:
```suggestion
	folder: Folder,
```
Calling the folder `context` inside the context object seems confusing to me 😵

We should use "context" only for context objects

Reply:
it's to do a clear separation between the nodes.